content/articles/understanding-blockchain-as-a-service/index.md

```diff
@@ -0,0 +1,87 @@
+### Blockchain as a Service
+
+### Introduction
+The blockchain is the new buzzword in town and everyone seems to be talking about it. But how do businesses actually use the technology without having to build their own blockchains? Blockchain as a service (BaaS) enables companies to develop applications on various blockchains— such as Ethereum, Ripple or NEO —without having to code from scratch. Clients benefit from BaaS because it allows for faster application development, lower maintenance costs, and faster adoption of Blockchain technology.
+### Topic to handle
+- Defining Baas.
+- Why choose Baas.
+- What to check for while choosing Baas.
+- Providers of Baas.
+
+### Prerequisites
+To understand this article one should have:
+1. Good knowledge and understanding of how blockchain works.
+   
+### Definition of BaaS.
+Baas involves customers using cloud services to create, host, develop and run their blockchain applications, and smart contracts in the software and any other relevant operations that can run blockchain technology. The cloud services are offered and managed by the cloud services provider. They are also responsible for any other related task to keep the systems up and running.
+
+This is an introduction to BaaS, an important turning point in the blockchain ecosystem, with the potential to drive the widespread use of distributed ledger technologies industries. These are founded on and operates on principles that are alike to those used in SaaS. 
+
+###Why choose BaaS?
+Blockchain technology is increasingly being investigated by IT organizations across a wide range of industries for its strategic benefits. There are some technical difficulties, absence of domain expertise, and operational expenses associated with establishing, managing, and maintaining the Blockchain, as well as maintaining the infrastructure, typically lead key decision-makers to take their time with adoption plans.
+
+1. Businesses that use Blockchain as a Service acquire access to qualified Blockchain developers, procedure and management specialists, the whole infrastructure from the cloud service provider for developing, programming, and deploying, not worrying about budget costs for any other operation that would cost them more.
+
+2. In most situations, a reputable BaaS partner is a source of practical expertise and knowledge that can improve the security of the systems. It considerably reduces the dangerous cases that might otherwise be encountered if they were created in-house.
+
+### Understanding BaaS
+BaaS allows users to create, host, and administer their blockchain apps and related blockchain processes using cloud-based technology. The infrastructure is kept adaptive and operating by the cloud-based service provider.
+BaaS is considered as an exploit of blockchain in the blockchain ecosystem that will help enterprises adopt blockchain technology.
+### How it Works
```

Comment:
This topic doesn't seem to talk much about "How it works?"

These points talk about the roles and responsibilities of a BaaS provider.

Instead, if you compare and contrast with SaaS, the reader would be able to differentiate it well